Cargo.toml

```diff
@@ -111,3 +111,13 @@ strip = false
 debug-assertions = false
 overflow-checks = false
 lto = true
+
+[package.metadata.deb]
+assets = [
+   ["debian/apt-cacher-rs.conf", "/etc/apt-cacher-rs/apt-cacher-rs.conf", "644"],
+   ["target/release/apt-cacher-rs", "/usr/sbin/apt-cacher-rs", "755"],
```

Comment:
Why is `apt-cacher-rs` installed into */usr/sbin* and not */usr/bin*, since it does not need root privileges?